R/github-actions.R

```diff
@@ -160,12 +161,12 @@ use_github_action <- function(name,
   save_as <- path(".github", "workflows", save_as)
   create_directory(path_dir(proj_path(save_as)))
 
-  new <- write_over(proj_path(save_as), contents)
+  new <- write_over(proj_path(save_as), contents, overwrite = overwrite)
 
   if (open && new) {
     edit_file(proj_path(save_as))
   }
-  if (!is.null(readme)) {
+  if (!is.null(readme) && !overwrite) {
```

Comment:
This is not 100% correct but is expedient, and I don't think another argument is worth it.